source/images/ehr_launch_seq.txt

```diff
@@ -0,0 +1,18 @@
+title EHR App Launch Flow
```

Comment:
Is this file needed for anything ? Seems like its in the images folder and its a text file. 
Does everything work if we remove this file out ?

Reply:
Seems like this is the text that the ehr_launch_seq.png shows up. I tried in my own forked repo to remove these two txt files and it looks like the site could still work fine.